fvm/state/view.go

```diff
@@ -0,0 +1,21 @@
+package state
+
+import (
+	"github.com/onflow/flow-go/model/flow"
+)
+
+type View interface {
+	NewChild() View
+	MergeView(child View)
+	Ledger
+}
```

Comment:
@m4ksio is this the right place to define this interface type?

Reply:
Looks right to me